sklearn/metrics/_plot/roc_curve.py

```diff
@@ -160,6 +160,12 @@ def plot_roc_curve(estimator, X, y, *, sample_weight=None,
     display : :class:`~sklearn.metrics.RocCurveDisplay`
         Object that stores computed values.
 
+    See also
```

Comment:
```suggestion
    See Also
```

Reply:
@glemaitre Thanks so much for reviewing!

The reason I used lowercase "also" is because that's what the [contributing guide](https://scikit-learn.org/dev/developers/contributing.html#guidelines-for-writing-documentation) suggested, and other documentation pages I reviewed also used a lowercase "also".

So sorry if this is a dumb question, but can you help me to understand when "also" should be lowercase and when it should be uppercase? Thank you!

Reply:
Basically it should always be capitalized. However, we are transitioning to validate the documentation with numpydoc so we have a lot of historical inconsistencies both in the source code and in the guideline.

The reference is here: https://numpydoc.readthedocs.io/en/latest/format.html (Section 12).

Reply:
@glemaitre Thanks for the info and the reference link - that's very helpful!

I committed your suggested edit.